cogs/ahk/ahk.py

```diff
@@ -295,24 +296,50 @@ async def cloudahk_call(self, ctx, code, lang='ahk'):
 		stdout, time = result['stdout'].strip(), result['time']
 
 		stdout = stdout.replace('`', '`\u200b')
+		file = None
+		encoded_out = stdout.encode('utf-8')
+		if len(stdout) < 1800 and stdout.count('\n') < 20 and stdout.count('\r') < 20:
+			#upload as plaintext
+			valid_response = 'No output.\n' if stdout == '' else '\n```autoit\n{0}\n```'.format(stdout)
+		elif len(encoded_out) < 100000 and AHK_GUILD_ID == ctx.guild.id:
```

Comment:
What's the significance of 100000?

Reply:
A limit of 100kb to the pastebin.
```suggestion
		elif len(encoded_out) < 100000 and AHK_GUILD_ID == ctx.guild.id: # 100kb
```